apps/vr-tests-react-components/src/stories/Menu/ScrollableMenuSmallViewport.stories.tsx

```diff
@@ -0,0 +1,82 @@
+import * as React from 'react';
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28622&commitId=29b5b69fd16cd4292e3d0b9a1aea1a4990958f04&env=prod&clientType=FLUENTUI)


  <details><summary><b>Menu</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Menu.Scrollable Menu Small Viewport.default.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

Reply:
Verified

Reply:
There should be on hover, let me try to add hover in test so scrollbar is visible

Reply:
Just pasting the playwright link you sent to me here for anyone else happens to see this: https://github.com/microsoft/playwright/issues/5778#issuecomment-796264504